codegenerator/cli/src/hbs_templating/codegen_templates.rs

```diff
@@ -454,7 +444,6 @@ impl EventTemplate {
     }
 
     pub fn from_config_event(config_event: &system_config::Event) -> Result<Self> {
-        let name = config_event.name.to_capitalized_options();
```

Comment:
Stopped using `to_capitalized_options` since we use only `capitalize`, I think it's better to have it only in the format to prevent using uncapitalized version by mistake.